src/main/java/org/ays/common/util/validation/EmailAddressValidator.java

```diff
@@ -39,6 +41,41 @@ public boolean isValid(String email, ConstraintValidatorContext constraintValida
         if (!StringUtils.hasText(email)) {
             return true;
         }
-        return email.matches(EMAIL_REGEX);
+
+        email = email.trim();
+        if (email.startsWith(" ") || email.endsWith(" ")) {
+            return this.buildViolation(constraintValidatorContext, "email must not start or end with whitespace");
+        }
```

Comment:
Bu kisim dogru calismiyor gibi gorunuyor. Trimledikten sonra bastaki ve sondaki space'ler gidecegi icin her zaman false olacak. Bu kisimla alakali test case eklenmedigi icin gozden kacmis sanirim. Burayi ve testleri guncelleyebilir miyiz

Reply:
doğru çalışıyor aslında  hata mesajı döndürebilsin diye bu kontrolü eklemiştim

Reply:
burada if'in icine hangi kosulda girecek

Reply:
@MenekseYuncu boşluklu gönderip kaydettiğimde database'den hata dönüyor gibi görünüyor. Postman üzerinden test etmiş miydik?
![image](https://github.com/user-attachments/assets/98c1b1ab-fb58-49b6-bc1e-d4b4d0eeb21d)

Reply:
@agitrubard ettim hatta @filizhelvaci  ile beraber etmiştik garip :'  bu akşam müsait olursam tekrardan test ederim